tsconfig.base.json

```diff
@@ -9,7 +9,7 @@
     "declaration": true,
     "declarationMap": true,
     "sourceMap": true,
-    "jsx": "react-jsx",
+    "jsx": "react",
```

Comment:
For my own edification, what's the difference here?

Reply:
The new JSX transform is only introduced in newer versions of React. Setting `react-jsx` here causes errors because the compiler tries to use `react/jsx-runtime` to do the JSX transform, which isn't available for React 16.8. Changing this to `react` tells the compiler to not use that transform and just convert JSX to `React.createElement` calls